cmd/controller/main.go

```diff
@@ -166,7 +166,7 @@ func main() {
 			configurationInformer,
 			revisionInformer,
 			coreServiceInformer,
-			virtualServiceInformer,
```

Comment:
I notice in this same file that `virtualServiceInformer` is still defined and synced. Can't both of those goes away?

Reply:
Nevermind. It's still used by the cluster ingress controller.

Reply:
However... shouldn't the cluster ingress controller _not_ be started by this controller manager?

If the end-game is decoupling from Istio, then I assume operators deploying knative serving should choose their own cluster ingress controller in the same way they'd have chosen their own (plain old) ingress controller with plain k8s.

Reply:
Maybe that happens in a follow-up?

Reply:
@krancour That's a later increment.

Reply:
good point. We'll have a special work to decide the shape of underlying network programming controllers after this.